### PR TITLE
Add switch to disable removing host

### PR DESCRIPTION
**Pull request by live1206**
Temp workaround for [AB#230323](https://ceapex.visualstudio.com/Engineering/_workitems/edit/230323/)

This PR will be reverted when partner implement the feature in another way.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6173)

### CODE REVIEW

File: src/docfx/config/Config.cs [hunk 1]
```diff
@@ -300,6 +300,12 @@ internal class Config : PreloadConfig
         /// </summary>
         public bool SkipMonikerValidation { get; private set; }
 
+        /// <summary>
+        /// Determines if remove host for external link.
+        /// </summary>
+        ///
```
Comment (OsmondJiang) [inline src/docfx/config/Config.cs:306]: remove

Comment (superyyrrzz) [inline src/docfx/config/Config.cs:306]: remove line

File: src/docfx/config/ops/OpsConfig.cs [hunk 2]
```diff
@@ -11,6 +11,9 @@ namespace Microsoft.Docs.Build
     [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
     internal class OpsConfig
     {
+        // TODO: remove this switch
```
Comment (superyyrrzz) [inline src/docfx/config/ops/OpsConfig.cs:14]: Add the task link here? We can better know when we can remove this.

https://ceapex.visualstudio.com/Engineering/_workitems/edit/230323/

Comment (live1206) [reply src/docfx/config/ops/OpsConfig.cs:14]: Prefer to revert this PR when the work item is ready

**Pull request closed without merging**